roofit/roofit/src/RooDSCBShape.cxx

```diff
@@ -0,0 +1,256 @@
+/*****************************************************************************
+ * Project: RooFit                                                           *
+ * Package: RooFitModels                                                     *
+ * @(#)root/roofit:$Id$
+ * Authors:                                                                  *
+ *    T. Skwarnicki modify RooCBShape to Asymmetrical Double-Sided CB                     *
+ *    Michael Wilkinson add to RooFit source                                 *
+ *****************************************************************************/
+
+/** \class RooCBShape
+    \ingroup Roofit
+
+PDF implementing the Asymmetrical Double-Sided Crystall Ball line shape.
```

Comment:
Could we add a latex equation here? See e.g.
https://github.com/root-project/root/blob/master/roofit/roofit/src/RooJohnson.cxx#L25

If you also happen to have a plot, I would add this once this is merged.

Reply:
done. I can create a plot after this is merged.